api/AmbientLightSensor.json

```diff
@@ -49,7 +49,9 @@
             "chrome": {
               "version_added": "56"
             },
-            "chrome_android": "mirror",
+            "chrome_android": {
+              "version_added": false
```

Comment:
The parent feature has flag data, so I think this might be a case of https://github.com/mdn/browser-compat-data/issues/6509. In one way of interpreting the previous data it means that the `AmbientLightSensor` constructor was supported in Chrome Android 56, behind a flag. (The other interpretation would say the data is inconsistent.)